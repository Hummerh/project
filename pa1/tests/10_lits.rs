b'\'' b'\\' b" \"\\\" "
1_2_3_4
456_789
