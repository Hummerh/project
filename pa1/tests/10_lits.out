LITCHAR(')
LITCHAR(\)
LITSTR( "\" )
LITDEC(1234)
LITDEC(456789)
